ni_measurementlink_service/session_management/_types.py

```diff
@@ -287,11 +321,27 @@ class Connection(NamedTuple):
     session_info: SessionInformation
     """The instrument session information."""
 
+    multiplexer_resource_name: Optional[str]
```

Comment:
`Optional[str]` only affects typing and does not make the constructor paraemter optional. To do that, add a default value, like `multiplexer_resource_name: Optional[str] = None`.

Reply:
Yes, I've removed "Optional" as the connection construction occurs with the `multiplexer_resource_name` and `multiplexer_route`.

I've retained `multiplexer_session_info` as `Optional[MultiplexerSessionInformation]` so that it can hold _None_.